### PR TITLE
Seeds for db

### LINKED ISSUE #154: Deployment: Ensure App Deployment - develop branch for Milestone 1
## User story

**Who: As a** developer
**What: I want to be** able to see the development version online on https://staging-simply-name-it.herokuapp.com/
**Why: so that I can** test it in **DEVELOPMENT** before is goes to Production

## Acceptance criteria

Make sure that every **PR builds** on Develop and that the **Develop Heroku DB** is up to date and with the correct credentials, and both Migrations and Seeding are updated in this environment. 

image

## Implementation details

Check the env variables on https://dashboard.heroku.com/apps/staging-simply-name-it/settings and **DO NOT SHARE or COMMIT any information about Users or Passwords.**

### PR DESCRIPTION
**Pull request by NatsGt**
# Description

Added a board past the deadline

Fixes # (issue)

# How to test?

npm run db:setup

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [x] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [x] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/server/seeds/development/05_ballots.js [hunk 1]
```diff
@@ -77,6 +77,24 @@ exports.seed = function (knex) {
...
+        },
+        {
+          boardId: 2,
+          userId: 95,
+          candidateId: 5,
+          rank: 2,
+        },
+        {
+          boardId: 2,
+          userId: 75,
```
Comment (dpfernandes) [inline src/server/seeds/development/05_ballots.js]: Can you change this to 95?

**Review by dpfernandes — CHANGES_REQUESTED** [review]

File: src/server/seeds/production/05_ballots.js [hunk 2]
```diff
@@ -77,6 +77,24 @@ exports.seed = function (knex) {
...
+        },
+        {
+          boardId: 2,
+          userId: 95,
+          candidateId: 5,
+          rank: 2,
+        },
+        {
+          boardId: 2,
+          userId: 75,
```
Comment (dpfernandes) [inline src/server/seeds/production/05_ballots.js]: 95 here as well on the userId

**Review by dpfernandes — CHANGES_REQUESTED** [review]

**Review by dpfernandes — APPROVED** [review]
👍

**Comment by senner007** [comment]
@NatsGt @dpfernandes 
The build failed. Check the error

**Pull request merged**